portfolio/src/main/webapp/career.html

```diff
@@ -155,13 +155,12 @@ <h2 class="section-title">Clubs and Student Organizations</h2>
                </form>
             </div>
 
-            <!-- TODO: edit style for user-login and user-logout divs -->
             <div id="user-login"> 
-                <p>To add a comment <a href="/user" onclick="checklogin()">Log in</a></p>
+                <p>To add a comment&nbsp;&nbsp;<button id="login-button" onclick="login()">Log in</button></p>
             </div>
 
             <div id="user-logout"> 
-                <a id="user-logout-link" href="">Log out</a>
+                <a id="logout-button" href="">Log out</a>
```

Comment:
Nice job! I would just add a todo that you'll implement an action for logout-button soon.

Reply:
Logout function also works. Forgot to write in description

Reply:
Oh. I don't see this element connected to a logout function or API at all, so how does it work?

Reply:
UserServlet generates logoutUrl (line 25) which is passed through json object to JS promise which outputs it to the page. So when user is logged in logout-button is shown and href is changed to a link which logs user out.